api/end_device.proto

```diff
@@ -68,10 +68,8 @@ message MACParameters {
   uint32 adr_tx_power_index = 5 [(gogoproto.customname) = "ADRTxPowerIndex", (validate.rules).uint32.lte = 15];
   // ADR: number of retransmissions.
   uint32 adr_nb_trans = 6 [(gogoproto.customname) = "ADRNbTrans", (validate.rules).uint32.lte = 15];
-  // ADR: number of messages to wait before setting ADRAckReq.
-  uint32 adr_ack_limit = 7 [(gogoproto.customname) = "ADRAckLimit", (validate.rules).uint32 = {gte: 1, lte: 32768}];
-  // ADR: number of messages to wait after setting ADRAckReq and before changing TxPower or DataRate.
-  uint32 adr_ack_delay = 8 [(gogoproto.customname) = "ADRAckDelay", (validate.rules).uint32 = {gte: 1, lte: 32768}];
+  reserved 7; // Deprecated: adr_ack_limit
```

Comment:
This breaks DB compatibility for already stored end devices, right?

Reply:
No. Why?
We do lose the old values, but all of these are default values from band, which, in fact, are all equal to the same value.
In order to keep the behavior the same a possibility is to wrap the ADRAck*Exponents in structs to distinguish between them being set and not and if not set, use the default values from band.

Reply:
If we lose data, then I'd say this counts as breaking DB compatibility.

I don't think it's a big issue, because there was no way for users to set these fields to anything other than the defaults, but still.